src/main/Entities/GetReview.java

```diff
@@ -0,0 +1,9 @@
+package Entities;
+
+import java.util.ArrayList;
+
+public class GetReview{
```

Comment:
we should probably expand this into GetReview.fromUser, and GetReview.fromRecipe in the future (each that take in the UserID or RecipeID) as we'll have 2 different places to view and store recipes.

but this is a good start to the class now so i'll merge it!